astropy/timeseries/tests/test_downsample.py

```diff
@@ -27,6 +29,14 @@
 )
 
 
+def assert_masked_equal(a, b):
```

Comment:
It might be out of scope. This helper is similar (but functionally not identical) to:

https://github.com/astropy/astropy/blob/72a1d42bd395caf07bc4ff1b1ba7312c99fdbfd3/astropy/utils/masked/tests/test_masked.py#L22-L26

Reply:
I probably should unify that one and the one introduced here into `astropy.tests`. I think it is a bit out of scope here, but am happy to be convinced otherwise.